test/image/test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+set -e
+
+GOPATH=/home/go
+export GOPATH
+
+WEAVE_SRC=$GOPATH/src/github.com/weaveworks/weave
+
+# Expect that the weave repo will be bind-mounted into $GOPATH
```

Comment:
I'm not sure this check is needed here.  It's done in the build container because part of the point of that is to make it easier for people outside of weaveworks to build weave.  But it looks like this container image will be used in a much more particular context, and won't be documented for outside use.